### DISCUSSION TITLE
formily V2 支持服务端渲染吗？

### DISCUSSION THREAD

**Discussion by wwyx778** [alibaba/formily#2248] [Q&A 2.x] [answered]
formily V2 支持服务端渲染吗？

**Comment by janryWang** [comment, marked as answer]
支持

**Reply by huaxiangsirenbutong** [reply]
请问在next.js 使用中样式引入失败怎么解决。
报错： Can't resolve '@formily/antd/lib/array-cards/style/css' in '/Users/jiweiji/Desktop/.....
![image](https://user-images.githubusercontent.com/37055241/137428661-51782e5a-fe9a-4e53-ab29-e8455ac7fd21.png)
